src/style.css

```diff
@@ -1 +1,104 @@
 /* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    background-color: #fff;
+    position: relative;
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+    padding: 0 50px;
+  }
+}
+
+@media (max-width: 1200px) {
+  .header {
+    background-color: #fff;
+    position: relative;
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+    padding: 0 50px;
+  }
+}
+
+.nav {
+  display: block;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
+
+ul {
+  padding-inline-start: 40px;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-content: space-between;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14px;
+  margin-left: 20px;
```

Comment:
1) move margin-left: 20px; to `.nav__item` (after you add `<li>` to your html)
2) use `:not:(:first-child)` to add margin to all of your items besides first one

Reply:
isn't fixed